chain/vm/vm.go

```diff
@@ -545,10 +546,11 @@ func (vm *VM) ApplyMessage(ctx context.Context, cmsg types.ChainMsg) (*ApplyRet,
 		}
 	}
 
-	var errcode exitcode.ExitCode
 	var gasUsed int64
 
-	if errcode = aerrors.RetCode(actorErr); errcode != 0 {
+	errcode := aerrors.RetCode(actorErr)
+
+	if errcode != 0 {
```

Comment:
You can revert this change to reduce the diff size.

Reply:
done